tensorbay/client/segment.py

```diff
@@ -204,14 +203,13 @@ def _post_multipart_formdata(
         self,
         url: str,
         local_path: str,
-        remote_path: str,
         data: Dict[str, Any],
     ) -> None:
         with open(local_path, "rb") as fp:
             file_type = filetype.guess_mime(local_path)
             if "x-amz-date" in data:
                 data["Content-Type"] = file_type
-            data["file"] = (remote_path, fp, file_type)
+            data["file"] = ("", fp, file_type)
```

Comment:
The commit message should be updated due to this change not only affect oss.
also you should test it on other cloud storage platform (aws and azure)